libproxy/modules/config_w32reg.cpp

```diff
@@ -160,7 +161,10 @@ class w32reg_config_extension : public config_extension {
 		if (get_registry(W32REG_BASEKEY, "ProxyOverride", &tmp, NULL, NULL)) {
 			string po = tmp;
 			delete tmp;
-			if (po == "<local>")
```

Comment:
Do you know what the <local> was supposed to be handling before? Did that work from some situations? If we need to do the whole '; --> ,' thing, I'm not sure what was working before.

Reply:
Actually, this wasn't working before.  As mentioned here: https://github.com/libproxy/libproxy/issues/96

This library has not been extensively used in windows environments from what I can tell.